README.md

```diff
@@ -1,2 +1,37 @@
 # Hello_world
+
 my first repo
+
+## Built with
+-HTML
+-CSS
+
+>To get a local copy running,follow these steps
+
+## Prerequisites
+-code Editor
+-Git
+-Github Account
+
+## setup
+
+-git init
+-git clone https://github.com/yusufnviiri/Hello_world.git
+
```

Comment:
- [ ] I would recommend adding all the steps here after `cloning` this repository & code to get into that project folder. For example:-
```
git clone https://github.com/yusufnviiri/Hello_world.git
cd Hello_world
open index.html file to see the project
```